### ISSUE TITLE
`RbConfig::CONFIG['host_cpu']` returns `arm` on Apple Silicon (M1)

### ISSUE THREAD

**Issue by jlnwlf** [djberg96/sys-cpu#34]
## Context

`RbConfig::CONFIG['host_cpu']` returns `arm` on my M1 laptop (2021, macOS 12.4) instead of `arm64` (wild guess), as changed in #33 (#32).

```
> RbConfig::CONFIG['host_cpu']
=> "arm" 
```

https://github.com/djberg96/sys-cpu/blob/4a238b6df087e3dc263f5f17cc8bb3f634e8299b/lib/sys/darwin/sys/cpu.rb#L161

Which apparently doesn't work:

https://github.com/djberg96/sys-cpu/blob/4a238b6df087e3dc263f5f17cc8bb3f634e8299b/lib/sys/darwin/sys/cpu.rb#L177

```
undefined local variable or method `result' for Sys::CPU:Class
```

By changing the expected value to `arm` seems to do the trick...

## Config

- Ruby 2.6.5p114 (2019-10-01 revision 67812) [-darwin21]
- RVM 1.29.12

**Comment by djberg96** [comment]
Curious, I suspect this has something to do with Ruby 2.6.5.  I couldn't get 2.6.5 to install via rbenv, but I have 2.6.10. When I fired up 2.6.10 and checked `RbConfig::CONFIG['host_cpu']` it returned "aarch64", which is also valid.

**Comment by djberg96** [comment]
Fixed in 1.0.5, thanks for the report!
